libarchive.cabal

```diff
@@ -83,6 +84,9 @@ library
         unix-compat >=0.1.2.1,
         deepseq >=1.4.0.0
 
+    if os(windows)
+        cc-options: -U__GCC_ASM_FLAG_OUTPUTS__
```

Comment:
This is the workaround mentioned in https://github.com/haskell/c2hs/issues/199#issuecomment-649476689